vacancies/traineeship/index.html

```diff
@@ -43,7 +43,12 @@ <h2>Software development studio with a strong focus on clever solutions to compl
                     
                 <h3><a id="vacancy-link" href="/vacancies">Vacancies @ Code Orange</a> &#9656; <a id="job-title" href=".">TRAINEESHIP SOFTWARE ENGINEERING</a></h3>
                 <hr>
+<p class="warning">We appreciate your interest in the trainee position at Nubis/Code Orange. We would like to inform you that the vacancy is now closed to general applicants.
```

Comment:
The contact form on this page is broken now as wel, right?

Reply:
Did not see they were on the vacancy pages as well. Updated the PR